test/k8sT/Nightly.go

```diff
@@ -448,7 +467,8 @@ var _ = Describe("NightlyExamples", func() {
 
 		})
 
-		It("Updating Cilium stable to master", func() {
+		It("K8sValidated Updating Cilium stable to master", func() {
```

Comment:
I understand that the string was updated so that this would be triggered during PRs, but this should be moved out of the file `Nightly.go` if we're going to run it during PRs. Given that we need to get this running as part of PRs ASAP, this isn't blocking, but please file an issue to change the location of this test to a separate file / another Ginkgo test which is more in-line with the current organization of the tests that we have.